packages/hagrid/hagrid/cli.py

```diff
@@ -2631,6 +2668,47 @@ def quickstart_setup(
         raise e
 
 
+def get_urls_from_dir(
+    repo: str,
+    branch: str,
+    commit: Optional[str],
+    url: str,
+) -> List[str]:
+    notebooks = []
+    if commit is not None:
+        gh_api_call = (
+            "https://api.github.com/repos/"
+            + repo
+            + "/git/trees/"
+            + commit
+            + "?recursive=1"
+        )
+    else:
+        gh_api_call = (
+            "https://api.github.com/repos/"
+            + repo
+            + "/git/trees/"
+            + branch
+            + "?recursive=1"
+        )
+    r = requests.get(gh_api_call)
```

Comment:
Can we have a check for `status_code` post this and print/raise an error if status_code is not 200.

Reply:
Yeah! On it!

Reply:
Done! Please resolve this after taking a look at: https://github.com/OpenMined/PySyft/pull/6686/commits/f1c85189b2a7145f8b50bb8e2730f43880d5b2d3